crates/sui-types/src/messages_consensus.rs

```diff
@@ -178,19 +176,6 @@ impl ConsensusTransaction {
             .unwrap_or_default()
     }
 
-    pub fn verify(&self, committee: &Committee) -> SuiResult<()> {
-        match &self.kind {
-            ConsensusTransactionKind::UserTransaction(certificate) => {
-                certificate.verify_signature(committee)
-            }
-            ConsensusTransactionKind::CheckpointSignature(data) => data.verify(committee),
-            // EndOfPublish and CapabilityNotification are authenticated in
-            // AuthorityPerEpochStore::verify_consensus_transaction
-            ConsensusTransactionKind::EndOfPublish(_)
-            | ConsensusTransactionKind::CapabilityNotification(_) => Ok(()),
-        }
-    }
```

Comment:
nice! i was wondering where was this used...

Reply:
wait, where do we move the verification to?